### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3: "Choose target" & "Available instruments" text in profiler launch are not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#413]
### Steps to Reproduce

1. Follow the Pre-requisite to launch the Xamarin profiler.
2. Verify the color contrast ratio of "Choose target" & "Available Instruments" text for the color contrast ratio.

### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1

### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 3.2:1
See the screenshots attached

### Screenshot
![1 profiler launch page](https://user-images.githubusercontent.com/22956130/41589436-2388b03c-73d2-11e8-942b-24ea37a6d429.png)

### Environment
macOS High Sierra OSX version: 10.13.4 
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
XamarinProfiler-MAC
Wipro
